_lab/lab01.md

```diff
@@ -211,7 +223,7 @@ The code above is how we define functions in python. We will work on defining ou
 On a very high level, the `def fToC(fTemp)` and `def cToF(cTemp)` is what we call a function <strong>signature</strong>. 
 * In the function signature of `def fToC(fTemp)`, the keyword `def` tells python we are defining a function,
 * `fToC` is the name of the function and 
-* `fTemp` is the function's parameter enclosed in parenthesis (note, there can be zero or more parameters for a function, but in this case there is only one). 
+* `fTemp` is the function's parameter enclosed in parenthesis (note, there can be zero or more parameters for a function, but in this case there is only one). You can think of parameters as variables that the function can use (note that calling fToC(32) is telling the computer to call fToC and set fTemp = 32).
```

Comment:
I like to add backticks to the names of variables and functions to have them stand out.